### ISSUE TITLE
VIN number log

### ISSUE THREAD

**Issue by girtsuzans** [jenavarro/obdlogger#35]
Hello! Awesome project. But is there any possibility to get VIN number? I see that there is PIDs according it in file \app\src\app\tab2\obdInfo.js.ts.
Maybe someone managed to get it work?

`//VIN
        {mode: obdinfo.modeVin, pid: "00", bytes: 4, name: "vinsupp0", description: "Vehicle Identification Number", convertToUseful4: obdinfo.bitDecoder, isDefault:false},
        {mode: obdinfo.modeVin, pid: "01", bytes: 1, name: "vin_mscout", description: "VIN message count", convertToUseful: obdinfo.convertVIN_count, isDefault:false},
        {mode: obdinfo.modeVin, pid: "02", bytes: 1, name: "vin", description: "Vehicle Identification Number", convertToUseful: obdinfo.convertVIN, isDefault:false}`

**Comment by jenavarro** [comment]
Yes, it works fine. This is assuming the car itself provides that value.

**Comment by girtsuzans** [comment]
thanks, but maybe there is an example - i can't figure out how to get it work - this looks very similar-> https://github.com/EricSmekens/node-bluetooth-obd/issues/23
